spritec-preview/src/main.rs

```diff
@@ -17,7 +17,7 @@ use std::time::Duration;
 use std::error::Error;
 use std::thread;
 
-use vek::{Mat4, Rgba};
+use vek::{Mat4, Vec3,Vec4, Rgba};
```

Comment:
Could you remove all the changes to this file? All of this code is being removed and modifying it will cause all kinds of merge conflicts for me on my GUI branch. You can still have all of these changes locally if you want, just please don't commit them. :)

You can revert a single file using the instructions [here](https://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git). Or if it's easier you can just copy/paste all the old contents of the file from the master branch on GitHub.